.github/workflows/dynniq_ec2.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       env:
         ImageOS: win19
       with:
+        ruby-version: 2.6
```

Comment:
any reason not to use 3.0? that's what i'm running locally (on mac)

Reply:
No reason. Let's try 3.0